README.md

```diff
@@ -1,24 +1,112 @@
-# README
+# NerdWallet
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A mobile web application where you can manage your budget by tracking your spending
 
-Things you may want to cover:
+![screenshot](app/assets/images/screenshot.png)
 
-* Ruby version
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- PostgreSQL
 
-* Database creation
+## Live Link
 
-* Database initialization
+[Heroku](https://infinite-citadel-92406.herokuapp.com)
 
-* How to run the test suite
+## Loom Presentation
 
-* Services (job queues, cache servers, search engines, etc.)
+[Presentation](https://www.loom.com/share/ed9e536ea3e9411c8145b4278d7a4c7f)
 
-* Deployment instructions
+## Getting Started
 
-* ...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rails](https://gorails.com/)
+
+### Setup
+
+- Make sure you have Ruby on Rails set up properly on your computer
+- Clone or download this repo on your machine
+- Enter project directory
+
+### Install
+
+```sh
+bundle install
+```
+
+### Database
+
+```sh
+# Create user
+sudo -u postgres createuser recipe_app -s
+
+# Create the database
+rails db:create
+
+## Apply migration
+rails db:migrate
+
+# Load the schema
+rails db:schema:load
+```
+
+### Run
+
+```sh
+rails s
+```
+
+### Test
+
+```sh
+rspec
+```
+
+### Troubleshoot
+
+```sh
+### Rspec failing
+RAILS_ENV=test rake db:reset
+```
+
+```sh
+### New Scaffold
+rails g scaffold_controller model
+```
+
+```sh
+### Undo migration
+rake db:migrate VERSION=0
+```
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.